data/json/recipes/recipe_food.json

```diff
@@ -3769,12 +3774,8 @@
     "batch_time_factors": [ 80, 1 ],
     "qualities": [ { "id": "COOK", "level": 1 } ],
     "tools": [ [ [ "surface_heat", 5, "LIST" ] ] ],
-    "proficiencies": [
-      { "proficiency": "prof_food_prep" },
-      { "proficiency": "prof_forage_cooking" },
-      { "proficiency": "prof_knife_skills" }
-    ],
-    "components": [ [ [ "acorns", 1 ] ] ]
+    "proficiencies": [ { "proficiency": "prof_food_prep" }, { "proficiency": "prof_forage_cooking" } ],
+    "components": [ [ [ "cooked_acorns", 1 ] ] ]
```

Comment:
```suggestion
    "components": [ [ [ "acorns_cooked", 1 ] ] ]
```

Reply:
Oh god, what a shameful mistake, thank you!